File: FLOW010.cpp
#include <stdio.h>

int main()
{
	int t;
	char ch;
	scanf("%d",&t);
	while(t--)
	{
		scanf(" %c",&ch);
		if(( ch == 'b') || ( ch == 'B' ))
			printf("BattleShip\n");
		else if(( ch == 'c') || ( ch == 'C' ))
			printf("Cruiser\n");
		else if(( ch == 'd') || ( ch == 'D' ))
			printf("Destroyer\n");
		else if(( ch == 'f') || ( ch == 'F' ))
			printf("Frigate\n");
	}
	return 0;
}

File: chef/FLOW010.py
t = input()
while(t > 0):
	t -= 1
	s = raw_input()
	if(s == 'B') | (s == 'b'):
		print 'BattleShip'
	elif(s == 'C') | (s == 'c'):
		print 'Cruiser'
	elif(s == 'D') | (s == 'd'):
		print 'Destroyer'
	else:
		print 'Frigate'